src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ *      File name: Deadline.java
+ *      @author: Jerome Neo
+ *      Description: Deadline class that inherits from the Task class.
+ */
+public class Deadline extends Task {
+    protected String by;

```

Comment:
(personally idm this) but there are people who think that fields (especially non final ones) should always be private